kinto/integration-test/test_kinto_sync.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+import os
+import string
+import time
+import unittest
+
+from kinto_client import Client
+from kinto_client import replication
+
+
+class Kinto_Sync(unittest.TestCase):
+    """
+    Tests that given two Kinto servers you can successfully sync data
+    from one to the other
+    """
+
+    def setUp(self):
+        # Figure out what the IP address where the containers are running
+        f = os.popen('which docker-machine')
+        docker_machine = str(f.read()).strip()
+
+        if string.find(docker_machine, "/docker-machine") == -1:
+            print "Could not find docker-machine in our path"
+            exit()
+
+        f = os.popen('%s ip default' % docker_machine)
+        ip = str(f.read()).strip()
+
+        # Set our master and read-only end points and create our test bucket
+        self.master_url = "http://%s:8888/v1/" % ip
+        self.read_only_url = "http://%s:8889/v1/" % ip
+        self.credentials = ('testuser', 'abc123')
```

Comment:
Well, it seems to me some overhead to have one little file that just returns those credentials. The actual credentials don't matter, you just need a set in order to save anything to Kinto